packages/kbn-text-based-editor/src/text_based_languages_editor.tsx

```diff
@@ -147,6 +147,15 @@ export const TextBasedLanguagesEditor = memo(function TextBasedLanguagesEditor({
     useState<LanguageDocumentationSections>();
 
   const policiesRef = useRef<SerializedEnrichPolicy[]>([]);
+
+  // Registers a command to redirect users to the index management page
+  // to create a new policy. The command is called by the buildNoPoliciesAvailableDefinition
+  monaco.editor.registerCommand('esql.policies.create', (...args) => {
+    application?.navigateToApp('management', {
```

Comment:
So I'm a bit two mind here: if I type

```
from index | enrich _
```

at this point hitting enter will exit `Discover` and navigate to the Policy page. I'm not clicking and everything I've done so far (the ES|QL query) will be lost.

Perhaps a confirm modal would help? Or perhaps open it in another tab?
I'm all in for make it easy, but the transition feels too brutal to me.

Reply:
I hear it, I changed it to open in a new tab. I am not a fan of modals and the goal is to have a flyout so opening in a new tab is a good one for the initial release